src/puptoo/mq/consume.py

```diff
@@ -31,7 +31,5 @@ def init_consumer():
 
     consumer = Consumer(connection_info)
 
-    consumer.subscribe([config.ADVISOR_TOPIC,
-                        config.COMPLIANCE_TOPIC,
-                        config.MALWARE_DETECTION_TOPIC])
+    consumer.subscribe([config.ANNOUNCE_TOPIC])
```

Comment:
If you consume the announce topic, you're going to be getting *all* the messages across the platform. This is fine because you are checking for the 'service' key in app.py. The only issue is that it will be a bit slower than reading the actual kafka headers.

In order to get to the service key you are looking for in msg.get(), it has to open the message and read the JSON. Headers are accessible without reading in the whole message. It might be worth looking at changing that logic to look at headers rather than the message body